docs/authentication.md

```diff
@@ -32,7 +32,86 @@ client = BaseClient(token="<your_token>")
 
 By default a service account does not allows you to contact all LumApps API endpoints, to do so you need to get a token as a given user and then use this token to authenticate the requests
 
-The sdk BaseClient offers two methods to help with that `get_new_client_as` and `get_new_client_as_using_dwd` that allows you to get a new BaseClient correclty authenticated.
+<details>
+<summary>An example with curl</summary>
+<p>
+
+The flow is as follow:
+<br>
+
+<ol>
+    <li>
+        With your service account get a google access token
+        <div style="margin: 8px;">
+        To know how to get this token with curl and you service account follow <a href="https://gist.github.com/ryu1kn/c76aed0af8728f659730d9c26c9ee0ed"> this tutorial</a>
+        <br/>
+        For an extended documentation on that process you can follow the <a href="https://developers.google.com/identity/protocols/oauth2/service-account"> google documentation</a>
+        </div>
+    </li>
+    <li>
+        Use this token as the bearer token to call the <a href="https://apiv1.lumapps.com/#operation/User/Gettoken"> user/get endpoint</a>
+        <br/>
```

Comment:
Does not render correctly

Reply:
what do you mean by does not render correctly ? the link works so ...

Reply:
![image](https://user-images.githubusercontent.com/467711/97008293-ea6d7980-1542-11eb-96bf-59dc8cda2ad6.png)

Reply:
It renders correctly on my side when I go to the page, on firefox or chrome ....